### DISCUSSION TITLE
The `set-output` command is deprecated and will be disabled soon.

### DISCUSSION THREAD

**Discussion by dxvidmr** [alshedivat/al-folio#1496] [Q&A]
Hi! Everything was working normally and suddenly I get this error in the Actions section when I try to synchronize my changes.... Can anyone help me?

The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. 
For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Thanks!!

**Comment by george-gca** [comment]
Probably your repository is outdated. Try to update with the latest version.

**Reply by dxvidmr** [reply]
Ok, I'll try!
